src/coreComponents/functions/TableFunction.cpp

```diff
@@ -401,10 +402,12 @@ void TableFunction::outputTableData( OutputOptions const outputOpts ) const
 
 void TableFunction::initializePostSubGroups()
 {
+  m_writeInLog = isLogLevelActive< logInfo::TableLogOutput >( getLogLevel() );
```

Comment:
Not sure why we need this as a data member of the class. Looks like this is the only place it is used. Can't it just be a local variable here?